serialization-jackson/src/main/scala/org/apache/pekko/serialization/jackson/JacksonSerializer.scala

```diff
@@ -162,10 +162,10 @@ import pekko.util.OptionVal
       JacksonObjectMapperProvider(system).getOrCreate(bindingName, Some(new CBORFactory)))
 
 @InternalApi object Compression {
-  sealed trait Algoritm
-  object Off extends Algoritm
-  final case class GZip(largerThan: Long) extends Algoritm
-  final case class LZ4(largerThan: Long) extends Algoritm
+  sealed trait Algorithm
```

Comment:
I think we can just add a Algorithm trait and make Algoritm extends it, and in 2.0 we can do the migration.